### PR TITLE
block_with_write_threshold: Add case to test the write threshold

### PR DESCRIPTION
**Pull request by YongxueHong**
Add a case that test to change the write threshold for a block drive.

ID: 1731472
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

**Comment by YongxueHong** [comment]
@lolyu  @peixiu @kimi1978wy @zhenyzha  @qingwangrh pls help review, thanks.

**Comment by zhenyzha** [comment]
(1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.7.6.ppc64le.io-github-autotest-qemu.block_with_write_threshold: PASS (135.13 s)
 (1/1) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.block_with_write_threshold: PASS (187.31 s)

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Comment by peixiu** [comment]
Looks good to me and tested on win8-64 and win2019 guest passed.

 (1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.block_with_write_threshold.q35: PASS (133.76 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_with_write_threshold.q35: PASS (442.53 s)

Acked-by: Peixiu Hou phou@redhat.com

File: qemu/tests/block_with_write_threshold.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+           triggered.
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    def get_node_name(image_tag):
+        """ Get the node name. """
+        img_param = params.object_params(image_tag)
+        file = QemuImg(img_param, data_dir.get_data_dir(), image_tag).image_filename
```
Comment (lolyu) [inline qemu/tests/block_with_write_threshold.py]: better to use `storage.get_image_filename`

Comment (YongxueHong) [reply qemu/tests/block_with_write_threshold.py]: Thanks, have updated, pls help me to review it again.

**Comment by menli820** [comment]
Looks good to me and I tested on win10-64 and win2019 guest passed.

(1/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win10.x86_64.io-github-autotest-qemu.block_with_write_threshold.q35: PASS (235.24 s)
 (2/2) Host_RHEL.m8.u2.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.block_with_write_threshold.q35: PASS (149.92 s)

**Pull request merged**